nuget.config

```diff
@@ -12,6 +12,7 @@
 
     <!-- a local directory to allow testing nupkg files without pushing to a remote feed -->
     <add key="packagestore" value="packagestore" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />

```

Comment:
Going to need someone with permissions to add the ComputeSharp nuget packages to an approved feed.